testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -505,6 +507,35 @@ def test_fasta2select_ClustalW(self, tmpdir):
         assert len(
             sel['mobile']) == 23090, "selection string has unexpected length"
 
+    def test_fasta2select_nogap(self, tmpdir):
+        """test align.fasta2select() on aligned FASTA with no gap (Issue #3124)"""
+        try:
+            reference = mda.Universe(self.ref).select_atoms('all')
+            target = mda.Universe(self.tgt).select_atoms('all')
+            # Rename histidine and remove alt conformation to permit alignment
+            for residue_reference in reference.residues:
```

Comment:
Do these non-standard residue types even exist in these PDB files? i.e. is this code necessary?

Reply:
Ya, that code wasn't necessary. I removed it.